index.d.ts

```diff
@@ -458,7 +458,7 @@ declare namespace mercurius {
             ) => void
           ) => void;
           context?: (
-            connection: SocketStream,
+            sockeet: WebSocket,
```

Comment:
Is this the correct spelling?

```suggestion
            socket: WebSocket,
```